cvat/apps/engine/migrations/0087_job_parent_job_task_consensus_replicas_and_more.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 4.2.17 on 2025-01-13 15:51
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+import cvat.apps.engine.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("engine", "0086_profile_has_analytics_access"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="job",
+            name="parent_job",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="children_jobs",
```

Comment:
It should either be `children` or `child_jobs` - pluralizing both words in ungrammatical.

Reply:
Renamed to `child_jobs`.